src/components/Projects/ProjectStatusCardWrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import useProjectUpdates from '../../hooks/useProjectUpdates'
+import { ProjectStatus } from '../../types/grants'
+import { Project } from '../../types/proposals'
+import { ProjectHealth, UpdateAttributes } from '../../types/updates'
+import locations from '../../utils/locations'
+
+import ProjectStatusCard from './ProjectStatusCard'
+
+interface Props {
+  project: Project
+}
+
+function prioritizedStatus(
+  project: Project,
+  update?: UpdateAttributes
+): { status: ProjectStatus | ProjectHealth; date: Date; href?: string } {
+  const { status: projectStatus, updated_at, created_at } = project
+
+  const defaultStatus = { status: projectStatus, date: updated_at || created_at }
+
+  if ([ProjectStatus.Finished, ProjectStatus.Revoked, ProjectStatus.Paused].includes(projectStatus)) {
```

Comment:
Can we give a name to these group of statuses? What do they represent?

Reply:
not ongoing? vs the in progress/pending statuses

Reply:
"Ongoing” is shown but has the lowest priority, that's why it is not included there